src/python_testing/TC_AVSM_2_1.py

```diff
@@ -264,20 +284,23 @@ async def test_TC_AVSM_2_1(self):
                 endpoint=endpoint, cluster=cluster, attribute=attr.AllocatedVideoStreams
             )
             logger.info(f"Rx'd AllocatedVideoStreams: {value}")
+            asserts.assert_is_not_none(value, "AllocatedVideoStreams is None")
```

Comment:
This also needs to verify that values of the fields within the list of structs

Reply:
Thank you for review. Since video stream is not allocated, the returned list in empty, how should we verify the struct?